.github/workflows/release.yml

```diff
@@ -185,6 +185,28 @@ jobs:
         chmod 0600 ./key
         ./install/tool/ssh-add_expect ./key
         bash install/aur/deploy.sh
+    - name: Windows Installer
+      if: startsWith(github.ref, 'refs/tags/v')
+      env:
+        P_DIR: ${{ steps.prep.outputs.P_DIR }}
+        ARCH: x64
+      run: |
+        mkdir -p /tmp/NSIS/v2rayA && cd /tmp/NSIS
+        cp $GITHUB_WORKSPACE/LICENSE $PWD/License.txt
+        cp $GITHUB_WORKSPACE/gui/public/img/icons/windows-icon-255x255.ico v2rayA/v2raya.ico
+        cp $GITHUB_WORKSPACE/gui/public/img/icons/windows-bitmap-255x255.bmp v2rayA/v2raya.bmp
+        cp $GITHUB_WORKSPACE/install/windows/v2raya.xml v2rayA/
+        cp $GITHUB_WORKSPACE/install/windows/v2raya.nsi $PWD
+        cp $P_DIR/v2raya_windows_x64_${{ steps.prep.outputs.VERSION }}.exe v2rayA/
+        wget https://github.com/v2fly/v2ray-core/releases/download/v4.44.0/v2ray-windows-64.zip
```

Comment:
Should we replace `v4.44.0` with `latest`? The `v2ray-core` is keep updating.

Reply:
It should be modified after the release of v5, but not now, I think.

Reply:
> It should be modified after the release of v5, but not now, I think.

Means that we don't follow the `v4.45.0`(if it exists)?

Reply:
@Gaojianli We could follow it by manual in this duration.